privacyidea/static/components/login/controllers/loginControllers.js

```diff
@@ -102,19 +102,24 @@ angular.module("privacyideaApp")
     $scope.myCountdown = "";
     // We save the previous State in the $rootScope, so that we
     // can return there
-    $rootScope.$on('$stateChangeSuccess',
-        function (ev, to, toParams, from, fromParams) {
-            //debug: console.log("we changed the state from " + from + " to " + to);
-            //debug: console.log(from);
-            //debug: console.log(fromParams);
-            //debug: console.log(to);
+    $transitions.onBefore({},
```

Comment:
We could limit the call to transitions with target 'token.details' so this doesn't get called on every transition.

Reply:
The ``previousState`` is used in one other location. We could also use it in other cases in the future. I want to keep it with no limits, to have the same functionality as before.

Reply:
This could be triggered just for `'token.details'` and `'token.challenges'` but i agree, there is probably not that much overhead calling this function on every transition.

Reply:
So this can be resolved, right?